libcudacxx/include/cuda/std/detail/libcxx/include/ccomplex

```diff
@@ -19,14 +19,4 @@
 
 #include <complex>
 
-#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
-#  pragma GCC system_header
-#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
-#  pragma clang system_header
-#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
-#  pragma system_header
-#endif // no system header
-
-// hh 080623 Created
-
```

Comment:
Question: This does not look like the result of clang-format. Why was this removed?

Reply:
Because this is a pure forwarding header, which does nothing else than include another one.

In that case we mostly just not use the header guard. At the same time, I am wondering whether that is good

Reply:
Alright, thanks!